app/Helpers/Util.php

```diff
@@ -392,17 +392,36 @@ public function checkParentCampChanged($all, $in_review_status, $liveCamp)
                 Support::removeSupport($all['topic_num'], $all['parent_camp_num'], $allChildSupporters);
             }
         } else */
+      
         if($all['parent_camp_num'] != $all['old_parent_camp_num']) {
                 $allParentCamps = Camp::getAllParent($liveCamp);
-                $allChildSupporters = Support::where('topic_num', $all['topic_num'])
+                $supporterNicknames = Support::where('topic_num', $all['topic_num'])
                     ->where('end', 0)
                     ->whereIn('camp_num', $allParentCamps)
                     ->pluck('nick_name_id');
-                if (sizeof($allChildSupporters) > 0) {
-                    foreach ($allParentCamps as $parentCamp) {
-                        Support::removeSupport($all['topic_num'], $parentCamp, $allChildSupporters, $all['camp_num']);
+
+                if (sizeof($supporterNicknames) > 0) 
```

Comment:
Please align `if(){` as in the line 396. Similar foreach in the below statements